src/shared/get-paths.js

```diff
@@ -12,12 +12,16 @@ module.exports = function getPaths (inventory) {
     // Don't create dirs for functions that don't already exist
     if (!existsSync(src)) return
     // Ok, here we go
-    let { config } = inv.lambdasBySrcDir[src]
-    let nodeModules = join(src, 'node_modules', '@architect')
-    let vendorDir = join(src, 'vendor')
-    // Allow opting out of shared/views via config.arc @arc
-    let path = config.runtime.startsWith('nodejs') ? nodeModules : vendorDir
-    paths[src] = path
+    let lambdae = inv.lambdasBySrcDir[src]
+    if (!Array.isArray(lambdae)) lambdae = [ lambdae ]
```

Comment:
Would this approach not attempt to re-hydrate the same directory multiple times? If it's an array, we can just grab what we need from the first item to run it just once.

Reply:
It wouldn't rehydrate as ultimately this module is just assembling the set of paths to hydrate into, so the `paths` object will get clobbered over, but your point stands: no need to iterate over the array. Will tweak.

Reply:
Actually maybe this isn't an issue, kinda hard to tell just looking at the diff.

Reply:
I removed the inner loop.